sphinx/source/conf.py

```diff
@@ -53,26 +61,17 @@
     'bokeh.sphinxext.collapsible_code_block',
 ]
 
-needs_sphinx = '1.8'
-
-rst_epilog = """
-.. |bokeh.models|       replace:: :ref:`bokeh.plotting <bokeh.plotting>`
-.. |bokeh.plotting|     replace:: :ref:`bokeh.plotting <bokeh.plotting>`
-.. |Color|              replace:: :py:class:`~bokeh.core.properties.Color`
-.. |DataSpec|           replace:: :py:class:`~bokeh.core.properties.DataSpec`
-.. |Document|           replace:: :py:class:`~bokeh.document.Document`
-.. |HasProps|           replace:: :py:class:`~bokeh.core.has_props.HasProps`
-.. |Model|              replace:: :py:class:`~bokeh.model.Model`
-.. |Property|           replace:: :py:class:`~bokeh.core.property.bases.Property`
-.. |PropertyDescriptor| replace:: :py:class:`~bokeh.core.property.descriptor.PropertyDescriptor`
-.. |PropertyContainer|  replace:: :py:class:`~bokeh.core.property.wrappers.PropertyContainer`
-.. |PropertyUnitsSpec|  replace:: :py:class:`~bokeh.core.properties.PropertyUnitsSpec`
-.. |field|              replace:: :py:func:`~bokeh.core.properties.field`
-.. |value|              replace:: :py:func:`~bokeh.core.properties.value`
-"""
+needs_sphinx = '3.5'
+
+rst_epilog = open("rst_epilogue.txt").read()
```

Comment:
For consistency's sake, how about we rename the file to `rst_epilog.txt` to match the variable name `rst_epilog`?

Reply:
I believe Sphinx will try to to parse it as a page in the docs if we do that, at least unless we move it up out of the `source` tree

Reply:
sure that seems good